src/bidiServer/WebSocketServer.ts

```diff
@@ -157,7 +156,16 @@ export class WebSocketServer {
         }
 
         const plainCommandData = message.utf8Data;
-        debugRecv(plainCommandData);
+        try {
+          const plainCommandDataPrettyPrint = JSON.stringify(
```

Comment:
No need to `strigify` it back I think, let the debug handle it.

Reply:
Very good, it even adds colors to the JSON output!